front/.eslintrc.js

```diff
@@ -74,7 +72,7 @@ module.exports = {
     'newline-per-chained-call': 'off',
     'no-array-constructor': 'error',
     'no-console': ['warn', { allow: ['warn', 'error'] }],
-    'no-duplicate-imports': 'off',
+    'no-duplicate-imports': 'error',
```

Comment:
@IvaKop do you have any idea on how I can assure we won't need `import/no-duplicates` anymore besides CI?

Reply:
The built-in `no-duplicate-imports` seems to work better; caught some errors that slipped through.

Reply:
Ah, that's great! 🚀